reader/tests/system/count/test_count.py

```diff
@@ -34,7 +35,16 @@
 
 
 def setup_data(connection, cursor, deleted=False):
-    for fqid, model in list(data.items()) + list(other_models.items()):
+    all_models = list(data.items()) + list(other_models.items())
+    max_pos = max(m["meta_position"] for _, m in all_models)
+    cursor.execute(
```

Comment:
why can't you just create one entry in the position table with the max_position?

Reply:
Because the models reference all positions, not just the max one, so all have to exists since we defined it as a foreign key in the events table.